src/test/resources/features/refapp/06-patient/EditPatientRelationship.feature

```diff
@@ -0,0 +1,16 @@
+Feature: EditRelationshipType
+
+  Scenario Outline editPatientRelationship:
```

Comment:
there's a syntax issue here, check existing features to know how to use scenario outline

Reply:
Resolved thanks , is this good to you now